replapi_it/user.py

```diff
@@ -1,9 +1,48 @@
+# -*- coding: utf-8 -*-
+import json
+
+import requests
+
+getCyclesQuery = """
+  karma
+"""
+headers = {
+  "X-Requested-With": "ReplAPI-IT Team",
+  "Referrer": "https://staging.replit.com/",
+}
+url = "https://staging.replit.com/graphql"
+
+
 class _User(object):
```

Comment:
With #13, this class is replaced. I'm sorry for the bad timing, but you're probably going to have to code this again. Even with this, I think it would be better to give a tutorial on how to grab karma instead of making an out-of-place function.

Reply:
Ah, ok, what should I recode, the class, function, ect.

Reply:
@BD103, I think it's best I would edit the #13 branch so nothing bad happens.

Reply:
That branch is merged, so just create your own.